test/test_mps.py

```diff
@@ -15,6 +15,7 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import itertools
+import yaml
```

Comment:
any package dependency like this might cause issues. Although it's yaml and should be fine but it can be cause of an issue. @Ronian526 can you check if there are any other tests in test/ folder which use Yaml package ?

Reply:
A bunch of files under test folder are using yaml: test/test_meta.py, test/test_namedtuple_return_api.py, and under tools/test/, we should be fine.